_proto/hapi/chart/metadata.proto

```diff
@@ -17,6 +17,10 @@ message Maintainer {
 //
 // 	Spec: https://k8s.io/helm/blob/master/docs/design/chart_format.md#the-chart-file
 message Metadata {
+	enum Engine {
+		// gotpl is the default engine
+		gotpl = 0;
```

Comment:
Awesome feedback, I will work on it.

The thing is that I am not sure if I should use an Enum here, because as you can see, I am not using it as the type for Engine in the Metadata Message. I am not using it in the Engine type because then the yaml parser complains because the the imported value is a string not an int32 (expected by enum)

So in the end I use the Enum just to store the different options and load them in the linter. 

Do you have any other suggestion on how to achieve my goal?